R/README.md

```diff
@@ -1,6 +1,16 @@
 # R on Spark
 
 SparkR is an R package that provides a light-weight frontend to use Spark from R.
+### Installing sparkR
+
+Libraries of sparkR need to be created in `$SPARK_HOME/R/lib`. This can be done by running the script `$SPARK_HOME/R/install-dev.sh`.
```

Comment:
I am running this on a machine with no X-server hence no R-studio, so this kind of functionality will be needed for users like me. 

Even for R-studio I feel sparkR needs to be compiled with the version of R >= 3.0